### PR TITLE
Enable softlockup panic on extratests

### PR DESCRIPTION
**Pull request by foursixnine**
See: https://progress.opensuse.org/issues/50345
Verification Runs: https://openqa.suse.de/tests/overview?distri=sle&build=foursixnine%2Fos-autoinst-distri-opensuse%237771&version=15-SP2

### CODE REVIEW

File: tests/console/early_test_setup.pm [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    assert_script_run('cat /proc/cmdline');
+    assert_script_run('sysctl kernel.softlockup_panic');
+    assert_script_run('sysctl -w kernel.softlockup_panic=1');
+    assert_script_run('sysctl -p');
+
+}
+
+sub run {
+    my ($self) = @_;
+    $self->select_serial_terminal;
```
Comment (foursixnine) [inline tests/console/early_test_setup.pm]: The problem with this is that we don't have a serial console for all the backends, will be looking at this soon.

**Comment by foursixnine** [comment]
Putting this back in the plate, VR to come

File: lib/bootloader_setup.pm [hunk 2]
```diff
@@ -739,6 +739,12 @@ sub specific_bootmenu_params {
         return " @params ";
     }
 
+    # Enable kernel.softlockup_panic, unless explicitly disabled
+    # See bsc#1126782
+    if (!get_var("SOFTLOCKUP_PANIC_DISABLED", 0)) {
+        push @params, "kernel.softlockup_panic=1";
+    }
```
Comment (okurz) [inline lib/bootloader_setup.pm:746]: maybe better

```perl
push @params, 'kernel.softlockup_panic=1' unless get_var('SOFTLOCKUP_PANIC_DISABLED', 0);
```

so single ticks where you don't want to evaluate variables and use the common post-condition style, especially with "unless", with the benefit of saving lines for readability.

Comment (foursixnine) [reply lib/bootloader_setup.pm:746]: I actually can't remember why I didn't do it like that back then

Comment (foursixnine) [inline lib/bootloader_setup.pm:746]:
```suggestion
    push @params, 'kernel.softlockup_panic=1' unless get_var('SOFTLOCKUP_PANIC_DISABLED', 0);
```

**Comment by foursixnine** [comment]
> please squash the commits

Only squashed one commit, rest can stay for history of changes.

Plus: Added the new variable to the readme

**Pull request merged**